saleor/settings.py

```diff
@@ -325,9 +325,9 @@ def get_bool_from_env(name, default_value):
 VATLAYER_ACCESS_KEY = os.environ.get('VATLAYER_ACCESS_KEY')
 VATLAYER_USE_HTTPS = get_bool_from_env('VATLAYER_USE_HTTPS', False)
 
-ACCOUNT_ACTIVATION_DAYS = 3
```

Comment:
Where in saleor is this actually used ? I am not sure what to write in the documentation for this variable.

Reply:
I think it's legacy